src/createServer.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const {
+  getUserById, getAllUsers, createNewUser, removeUser, updateUser,
+} = require('./users');
+const {
+  getAllExpenses, getExpense, createNewExpense, removeExpense, updateExpense,
+} = require('./expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express().use(cors());
+
+  app.users = [];
+  app.expenses = [];
+
+  app.getUser = (userId) => {
+    return app.users.find(user => user.id === +userId);
+  };
+
+  getAllUsers(app);
+  getUserById(app);
+  createNewUser(app);
+  removeUser(app);
+  updateUser(app);
+
+  getAllExpenses(app);
+  getExpense(app);
+  createNewExpense(app);
+  removeExpense(app);
+  updateExpense(app);
```

Comment:
it's not the best approach to creating routes and it's hard to get used routes at a glance